src/Controller/Categorias/CategoriasOperarController.php

```diff
@@ -15,6 +15,12 @@ class CategoriasOperarController extends AbstractController
     public function registrarCategoria(CategoriaService $categoriaService, Request $request): Response
     {
         $nome = $request->request->get("nome");
+
+        if(empty(trim($nome))){
```

Comment:
o empty já verifica se é string vazia

```suggestion
        if(empty($nome)){
```

Reply:
tem outros lugares